src/Juno/Consensus/Pure/Handle/RequestVoteResponse.hs

```diff
@@ -87,7 +85,7 @@ handle m = do
            (RequestVoteResponseEnv
             (JT._role s)
             (JT._term s)
-            (lastLogIndex')
+            lastLogIndex'
```

Comment:
Not sure if unnecessary brackets win out over OCD... this line may be indented by a single space in the future